src/app/shared/search/search-result.component.ts

```diff
@@ -111,6 +113,12 @@ export class SearchResultComponent implements OnChanges {
     return this.resultView.results.length && this.paginationMetadata.total_pages_count;
   }
 
+  hasDrafts(): boolean {
```

Comment:
same as above. The best place for this method would be on the SearchResultView itself